### PR TITLE
implement send_to_address

### PR DESCRIPTION
**Pull request by keraliss**
This pr attempts to implement send_to_address command for maker-cli

### CODE REVIEW

**Comment by mojoX911** [comment]
@keraliss the code looks good. Let us know if you managed to run the command manually. Then we will merge this.

File: src/maker/rpc/server.rs [hunk 1]
```diff
@@ -111,6 +115,34 @@ fn handle_request(maker: &Arc<Maker>, socket: &mut TcpStream) -> Result<(), Make
...
+
+            let tx = maker.get_wallet().write()?.spend_from_wallet(
+                fee,
+                SendAmount::Amount(amount),
+                destination,
+                &coins_to_send,
+            )?;
+
+            let calculated_fee_rate = fee / (tx.weight());
+            println!("Calculated FeeRate : {:#}", calculated_fee_rate);
```
Comment (Shourya742) [inline src/maker/rpc/server.rs:138]: Use logger for these... rest looks ok. Did it worked for you locally?

**Review by Shourya742 — CHANGES_REQUESTED** [review]

File: src/maker/rpc/server.rs (hunk 1 above)
Comment (keraliss) [reply src/maker/rpc/server.rs]: not yet, makerd seems broken to me right now, waiting for #291 to get merged, should work after that

**Review by mojoX911 — APPROVED** [review]
Ack

**Pull request merged**